homeassistant/components/alexa/capabilities.py

```diff
@@ -300,6 +300,10 @@ class Alexa(AlexaCapability):
     The API suggests you should explicitly include this interface.
 
     https://developer.amazon.com/docs/device-apis/alexa-interface.html
+
+    To compare current supported locales in Home Assistant
+    with Alexa supported locales, run the following script:
+    script/alexa_locales.py
```

Comment:
```suggestion
    python -m script.alexa_locales
```

If direct execution of the script is wanted, some execute rights need to be added and a header to associate

Reply:
In `const.py` there is also a list of locales BTW

Reply:
The list in const.py is that all a summary of all possible locales? The script doesn't check that currently. If we extend the script we can add a note there.

Reply:
Yes, I saw where it was used. I was wondering how it is computed.

Reply:
I am not sure how it is computed. Something tells me we implement `English` only.